package.json

```diff
@@ -10,7 +10,7 @@
     "start": "react-scripts start",
     "build": "GENERATE_SOURCEMAP=false react-scripts build",
     "test": "react-scripts test",
-    "format": "prettier 'src/**/*/*.[tj]s' 'cypress/**/*/*.[tj]s' --ignore-path ./.prettierignore --write",
```

Comment:
The suggested [format](https://prettier.io/docs/en/cli.html) to run the command is as follows (hence the switch between `--write` and the code path):

```sh
prettier [options] [file/dir/glob ...]
```

Also:

- I have dropped the `--ignore-path ./.prettierignore` since it is the [default](https://prettier.io/docs/en/cli.html#--ignore-path)
- I have merged the code paths using `{}`
- I have dropped the erroneous extra `*` in the path

Reply:
Awesome, looks great, thanks!